ArduPlane/mode_manual.cpp

```diff
@@ -6,7 +6,7 @@ void ModeManual::update()
     SRV_Channels::set_output_scaled(SRV_Channel::k_aileron, plane.roll_in_expo(false));
     SRV_Channels::set_output_scaled(SRV_Channel::k_elevator, plane.pitch_in_expo(false));
     output_rudder_and_steering(plane.rudder_in_expo(false));
-    float throttle = plane.get_throttle_input(true);
+    float throttle = rc().has_valid_input() ? plane.get_throttle_input(true) : 0.0;
```

Comment:
in plane we handle this with default values in radio.cpp, see control_failsafe(). This seems to be defeating that?